catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -553,7 +579,7 @@ public static boolean isNfcSharedPreferenceEnabled(Context context) {
 	public static void setNfcSharedPreferenceEnabled(Context context, boolean value) {
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_SHOW_NFC_BRICKS, value);
-		editor.commit();
+		editor.apply();
```

Comment:
`putBoolean()` returns the editor again, so you can just append the `.apply()` above